class/blackbox-exporter.yml

```
@@ -8,4 +14,13 @@ parameters:
       - input_paths:
           - blackbox-exporter/component/main.jsonnet
         input_type: jsonnet
-        output_path: blackbox-exporter/
+        output_path: blackbox-exporter/${_instance}
+      - output_path: blackbox-exporter/${_instance}/10_blackbox_exporter_helmchart
+        input_type: helm
+        output_type: yaml
+        input_paths:
+          - blackbox-exporter/helmcharts/blackbox-exporter
+        helm_values: ${blackbox_exporter:helmValues}
+        helm_params:
+          release_name: ${_instance}
```

With Kapitan 0.30 this should be `name` (see https://syn.tools/commodore/explanation/migrate-kapitan-0.29-0.30.html#_changes_to_fields_in_helm_params)

```suggestion
          name: ${_instance}
```